src/request_handler.rs

```diff
@@ -25,8 +25,8 @@ impl RequestHandler {
     }
   }
 
-  async fn response(mut self) -> Response<Body> {
-    match self.list_www().await.context("cannot access `www`") {
+  async fn response_without_errors(mut self, request: Request<Body>) -> Response<Body> {
```

Comment:
I kinda don't like this name, although don't feel super strongly about it.

I think I'd slightly prefer keeping this called `response`, and calling the fallible function that's currently called `response` `dispatch` instead, since it looks like it's responsible for doing routing. What do you think?

Reply:
Sounds good.